SwiftWisdom/Core/StandardLibrary/Numbers/Bool+Extensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Bool+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension Bool {
+    public init(ip_string: String) {
+        let cleaned = ip_string.lowercaseString
+        let truePrefixes = [
+            "t",
+            "y",
+            "1",
+        ]
+        
+        self = truePrefixes.contains { cleaned.hasPrefix($0) }
```

Comment:
Is it widely accepted that strings beginning with y, t, or 1 represent a true value? I've never heard that.